Gemfile

```diff
@@ -7,6 +7,8 @@ git_source(:github) { |repo| "https://github.com/#{ repo }.git" }
 # Bundler will treat runtime dependencies like base dependencies, and
 # development dependencies will be added by default to the :development group.
 gemspec
+gem 'sidekiq', '~> 6.3', '>= 6.3.1'
```

Comment:
Can we put these in the gemspec instead?We put the context dependent dependencies here and the ones we always want there.

Reply:
Yup